var/spack/repos/builtin/packages/star/package.py

```diff
@@ -29,12 +29,11 @@ class Star(Package):
     """STAR is an ultrafast universal RNA-seq aligner."""
 
     homepage = "https://github.com/alexdobin/STAR"
-    url      = "https://github.com/alexdobin/STAR/archive/2.6.1a.tar.gz"
+    url      = "https://github.com/alexdobin/STAR/archive/2.6.1b.zip"
 
-    version('2.6.1a', '1ebaac553809c20900e0b42aadf75cc6')
+    version('2.6.1a', '87ae031376c904be1e0692f582beda7b')
```

Comment:
I assume this checksum change is due to `.zip` rather than `.tar.gz`?

Reply:
Unsure but that's the sum

```
me@donut[~/repos/spack]$ md5sum 2.6.1a.tar.gz 
1ebaac553809c20900e0b42aadf75cc6  2.6.1a.tar.gz
```

When does Spack move to `shasum`?

Reply:
you can use it right now if you want. There are a number of packages that use sha256 right now